docs/TROUBLESHOOTING.md

```
@@ -0,0 +1,396 @@
+calendar-bridge sync-once -config config.yaml -dry-run
+```
+
+It validates the config, authenticates every account, fetches events, and
+reports exactly what a real pass would change — **without writing to any
+calendar**. Safe to run at any time, including while the daemon is running.
+
+---
+
+## `account not yet authorized`
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)